make.cmd

```diff
@@ -64,9 +67,9 @@ if not exist %LIBJAGGED_TEST_TARGET% ( mkdir %LIBJAGGED_TEST_TARGET% )
 cd %LIBJAGGED_TEST_TARGET%
 cmake %LIBJAGGED_TEST_SRC% -G%TARGET_PLATFORM% ^
  -DINCLUDE_LIBGIT2=%LIBGIT2_SRC%\include ^
- -DLINK_LIBGIT2=%LIBGIT2_TARGET%\%CMAKE_CONFIG% ^
+ -DLINK_LIBGIT2=%LIBGIT2_TARGET% ^
  -DINCLUDE_LIBJAGGED=%LIBJAGGED_SRC% ^
- -DLINK_LIBJAGGED=%LIBJAGGED_TARGET%\%CMAKE_CONFIG%
+ -DLINK_LIBJAGGED=%LIBJAGGED_TARGET%
```

Comment:
Please don't break the VS builds.  Do a test here for the presence of mingw if this is the right behavior there.